.github/workflows/codeql-analysis.yml

```diff
@@ -8,6 +8,13 @@ name: 'code-ql'
 on:
   schedule:
     - cron: '0 20 * * *'
+  push:
```

Comment:
I guess we don't need the scheduled one then, right?

Reply:
I thought it is good if we do not work on the repo, but I guess you are right and the state should be save when we do it on each PR.